### PR TITLE
ports/esp32/boards: Add Deneyap Boards.

### PR DESCRIPTION
**Pull request by ozgurbostan**
- Deneyap Kart 
- Deneyap Kart 1A
- Deneyap Kart 1A v2
- Deneyap Kart G
- Deneyap Kart Mini
- Deneyap Kart Mini v2

### CODE REVIEW

**Comment by ozgurbostan** [comment]
Dear @dpgeorge,

Our PR ha not been merged, I wonder whether there something that we need to do?

**Comment by DogushC** [comment]
Hi @dpgeorge, could you merge the PR?

**Comment by ozgurbostan** [comment]
Hi @dpgeorge,

Any problem with the PR? It's been 3 months since any progress. If any update you need, let me know please.

**Comment by ozgurbostan** [comment]
Hi @dpgeorge,

Any progress? It is 6 months and still no response, even no reply for the enquiries.

Could you please tell us what is wrong or else accept our pull request?

File: ports/esp32/boards/DENEYAP_KART/modules/deneyapkart.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+DAC1 = const(25)
+DAC2 = const(26)
+
+# RGB LED
+LEDR = const(3)
+LEDG = const(1)
+LEDB = const(4)
+
+# BUTTON
+GPKEY = const(0)
```
Comment (mattytrentini) [inline ports/esp32/boards/DENEYAP_KART/modules/deneyapkart.py:27]: There's no problem with configuring these pins this way but, just to let you know, #12010 added support for _named pins_. The idea is to use a `pins.csv` file to specify an _alias_ to a pin. So you could alias `SPI_MOSI` to `Pin(5)` and access it with `machine.Pin.board.SPI_MOSI` or `machine.Pin(5)`.

See [ARDUINO_NANO_ESP32](https://github.com/micropython/micropython/tree/master/ports/esp32/boards/ARDUINO_NANO_ESP32) for an example.

File: ports/esp32/boards/DENEYAP_KART/board.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    "images": [
+        "deneyapkart.jpg"
+    ],
+    "mcu": "esp32",
+    "product": "Deneyap Kart",
+    "thumbnail": "",
+    "url": "https://deneyapkart.org/",
+    "variants": {
+        "idf3": "Compiled with IDF 3.x"
+    },
```
Comment (mattytrentini) [inline ports/esp32/boards/DENEYAP_KART/board.json:28]: I think removing `variants` from `board.json` makes sense here; from v1.21 there will only be IDF v5.x builds.
```suggestion
```

File: ports/esp32/boards/DENEYAP_KART/board.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        "Breadboard friendly"
+    ],
+    "id": "dydk_mpv10",
+    "images": [
+        "deneyapkart.jpg"
+    ],
+    "mcu": "esp32",
+    "product": "Deneyap Kart",
+    "thumbnail": "",
+    "url": "https://deneyapkart.org/",
```
Comment (mattytrentini) [inline ports/esp32/boards/DENEYAP_KART/board.json:25]: If possible, I believe that linking directly to the specific board is preferred.

```suggestion
    "url": "https://magaza.deneyapkart.org/en/product/detail/deneyap-kart",
```

**Review by mattytrentini — COMMENTED** [review]
Most of my comments to `DENEYAP_KART` also apply to the other boards.

**Comment by pxsty0** [comment]
Hello @dpgeorge , what do we need to do to add the cards?

**Comment by pxsty0** [comment]
@mattytrentini

**Pull request closed without merging**